tensorflow/core/graph/mkl_layout_pass_test.cc

```diff
@@ -2345,6 +2345,29 @@ TEST_F(MklLayoutPassTest, NodeRewrite_Conv2D_Basic) {
             "DMT/_1->C:3");
 }
 
+// Test case for the Depthwise FWD pass
+TEST_F(MklLayoutPassTest, NodeRewrite_DepthwiseConv2dNative_Basic) {
+  InitGraph(
+      "node { name: 'A' op: 'Input'}"
+      "node { name: 'B' op: 'Input'}"
+      "node { name: 'C' op: 'DepthwiseConv2dNative'"
+      " attr { key: 'T'                value { type: DT_FLOAT } }"
+      " attr { key: 'data_format'      value { s: 'NCHW' } }"
+      " attr { key: 'use_cudnn_on_gpu' value { b: false } }"
```

Comment:
`DepthwiseConv2dNative` doesn't have the attribute `use_cudnn_on_gpu`. (See [its attribute page](https://www.tensorflow.org/api_docs/cc/struct/tensorflow/ops/depthwise-conv2d-native/attrs), versus Conv2D's [attribute page](https://www.tensorflow.org/api_docs/cc/struct/tensorflow/ops/conv2-d/attrs).) Maybe remove this line?